src/modules/party-external-rating/party-external-rating.module.ts

```diff
@@ -0,0 +1,21 @@
+import { DynamicModule, Module } from '@nestjs/common';
+
+import { PartyExternalRatingController } from './party-external-rating.controller';
+import { PARTY_EXTERNAL_RATINGS_PROVIDER_SYMBOL, PartyExternalRatingModuleOptions } from './party-external-rating.module-definition';
+
+@Module({})
+export class PartyExternalRatingModule {
```

Comment:
Sort of related to an above comment about controllers -- is this module maybe a bit too narrowly defined, as to only cover the party external rating? Would it maybe be clearer to have a `PartyModule` which contains several different controllers? (I'm really not sure right now what will end up shared between the controllers -- to determine how to make decisions like this 😅)

Reply:
Good point 🤔 I think we'll find out which decision as better as we write more of the party endpoints. I think we should aim to have the modules be "only as big as they need to be" so we should wait until we see a reason that parties and party external ratings should be in the same module